Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM continuumio/miniconda3:4.9.2
```

Comment:
All of our conda files now have the same first few steps:
1. FROM miniconda3
2. create and use workdir
3. Install a specific python version via conda

Perhaps this is a candidate for a multi-stage build so that we don't have to update our miniconda and python versions in multiple places?

Reply:
Created https://github.com/azavea/nasa-hyperspectral/issues/83